app/modules/user/edit_User/editUser.controller.js

```diff
@@ -0,0 +1,19 @@
+const editUserQuery = require('./editUser.query')
```

Comment:
Get user details API is missing, you need to get the user details first, then only edit that details can be performed right?